terraform/templates/ec2_user_data.sh

```diff
@@ -2,6 +2,9 @@
 # Copyright (c) The Libra Core Contributors
 # SPDX-License-Identifier: Apache-2.0
 
+export log_path="/data/libra/${host_log_path}"
```

Comment:
Will this override the configs in validator.tf and variable.tf?

Reply:
Nope it takes input from the template_file in `validators.tf` and renders it. https://github.com/libra/libra/blob/317966efb1007eb1afa9716e0b56c7508fc16727/terraform/validators.tf#L134

The rendered user_data template_file is shared by both `validators.tf` and `fullnodes.tf`

Reply:
logs can be in one of two places: /data/libra for persistent storage, or /data/<container name>/_data